.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### Description of change
+
+_Please provide description of change here_.
+
+### Pull Request check-list
+
+_Please make sure to review and check all of these items_:
+
+- [ ] Does `make test` (or `vcbuild test nosign`) pass with this change?
+- [ ] Is commit message formatted according to [CONTRIBUTING.md][0]
+- [ ] Regression test (or benchmark if this is a performance-related change) is
+  included (only required for code changes)
+- [ ] Documentation change or addition is included (if this change modifies
+  existing APIs, or introduces new ones)
+
```

Comment:
What about saying above that these items could be checked after the PR is open, and if some of them are not done yet - they could be checked upon completion?